src/reservations/reservations.controller.ts

```
@@ -0,0 +1,577 @@
+      status: 401,
+      description: '인증되지 않은 사용자',
+      type: ErrorResponseDto,
+      example: {
+        code: '1003',
+        message: '유효하지 않은 토큰입니다.'
+      }
+    })
+    async getReservations(@Query() query: GetReservationsQueryDto): Promise<GetReservationsResponseDto> {
+       
+      const mockReservations = [
+        {
```

ESLint

[error] Replace `········` with `······`

(prettier/prettier)